main/src/ca/uwaterloo/flix/language/phase/Parser.scala

```diff
@@ -1175,6 +1179,16 @@ class Parser(val source: Source) extends org.parboiled2.Parser {
       SP ~ Names.QName ~ SP ~> ParsedAst.Expression.QName
     }
 
+    def LowerCaseVariable: Rule1[ParsedAst.Expression] = {
```

Comment:
Do we need this? I think we can just use `Names.LowerCaseName`.

Reply:
That returns a Name.Ident i believe, thus the fuzz here

Reply:
Lets use Name.Ident then.

Reply:
but that just postpones the trouble no? then its more work later to evaluate this `Ident`